.env.example

```diff
@@ -7,7 +7,10 @@ SESSION_SECRET="super-duper-s3cret"
 # https://letsencrypt.org/docs/expiration-emails/
 LETS_ENCRYPT_ACCOUNT_EMAIL="nx@senecacollege.ca"
 
-PORT=8080
```

Comment:
Why remove this? Is this no longer needed?

Reply:
see my other comment.